src/pages/home/components/SocialMedia.js

```diff
@@ -3,24 +3,35 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faFacebook, faTwitter, faInstagram } from '@fortawesome/free-brands-svg-icons';
 
 import config from 'config/constants';
-import styles from '../assets/socialmedia.scss';
 
 const SocialMedia = () => {
   return (
-    <div className={styles.socialMedia}>
-      <div className={styles.iconWrapper}>
+    <div className="flex flex-row-reverse self-center md:self-auto">
+      <div className="group m-3 p-4 rounded-full border border-solid border-primary-light cursor-pointer hover:bg-primary-light transition transition-fast transition-property-all transition-timing-linear">
```

Comment:
While you are here, can you add the github repository to this list? Use this icon: https://fontawesome.com/icons/github-alt?style=brands and this url: `https://github.com/Coding-Coach/coding-coach`